backend/api/Services/RobotService.cs

```diff
@@ -28,5 +40,32 @@ public async Task<IEnumerable<Robot>> ReadAll()
         {
             return await _context.Robots.FirstOrDefaultAsync(robot => robot.Id.Equals(id, StringComparison.Ordinal));
         }
+
+        public async Task<Report> StartMission(Robot robot, string missionId)
+        {
+            var response = await _isarService.StartMission(missionId);
+            if (!response.IsSuccessStatusCode)
+            {
+                _logger.LogError("Coult not start mission");
+                throw new MissionException($"Coult not start mission with id: {missionId}");
```

Comment:
```suggestion
                throw new MissionException($"Could not start mission with id: {missionId}");
```

Reply:
Is necessary to both write to logger and throw an exception?

Reply:
Not necessary, agreed